src/logic/basic.lean

```diff
@@ -1599,4 +1605,9 @@ by by_cases h : P; simp [h]
 lemma ite_and : ite (P ∧ Q) a b = ite P (ite Q a b) b :=
 by by_cases hp : P; by_cases hq : Q; simp [hp, hq]
 
+lemma ite_ite_eq_of_ne {α β} [decidable_eq β] {a b c : α} {i j k : β} (h : j ≠ k) :
```

Comment:
Would `ite_eq_ite_of_ne` be better?

Reply:
I want to emphasize the left hand side (actually both sides) is a double `ite`; I actually intend that the `eq` in the name refers to the `i = j`/`i = k` conditions, but you may also interpret the name as an abbreviation from `ite_ite_eq_ite_ite_of_ne`.

Reply:
What about `ite_ite_comm_of_ne`?